UI/window-basic-settings-a11y.cpp

```diff
@@ -230,170 +236,110 @@ void OBSBasicSettings::on_choose1_clicked()
 	UpdateA11yColors();
 }
 
+void OBSBasicSettings::on_choose1_clicked()
+{
+	QColor color = GetColor(
+		selectRed,
+		QTStr("Basic.Settings.Accessibility.ColorOverrides.SelectRed"));
+	if (!color.isValid())
+		return;
+	selectRed = color_to_int(color);
+
+	on_choose_clicked_update();
+}
+
 void OBSBasicSettings::on_choose2_clicked()
 {
 	QColor color = GetColor(
 		selectGreen,
 		QTStr("Basic.Settings.Accessibility.ColorOverrides.SelectGreen"));
-
 	if (!color.isValid())
 		return;
-
```

Comment:
Please add these lines back. We typically avoid placing new statements directly under an un-enclosed `if` statement.